cmake/Config.cmake.in

```diff
@@ -4,8 +4,8 @@
 #  @CMAKE_PROJECT_NAME@_INCLUDE_DIRS - Include directories for @CMAKE_PROJECT_NAME@
 #  @CMAKE_PROJECT_NAME@_LIBRARIES    - Libraries for @CMAKE_PROJECT_NAME@
 
-set(gflip_INCLUDE_DIRS @CMAKE_INSTALL_PREFIX@/include/gflip)
+set(gflip_INCLUDE_DIRS ${CMAKE_CURRENT_LIST_DIR}/../../../include/)
```

Comment:
Do we have to remove `gflip` subfolder?

Would that still work?

All the cmake stuff in this project is a little mess IMHO. I couldn't even see where the install rule for the include files until looking into the whole tree:
https://github.com/clearpathrobotics/gflip/blob/master/src/gflip/CMakeLists.txt#L17

@afakihcpr Any concerns?